src/libraries/System.Private.CoreLib/gen/System.Private.CoreLib.Generators.csproj

```diff
@@ -5,6 +5,7 @@
   </PropertyGroup>
```

Comment:
Side note, `    <TargetFramework>netstandard2.0</TargetFramework>` this can probably use `$(NetCoreAppCurrent)` as it is only used by corelib building for current version.

Reply:
There may be cases where the generator runs on .NET Framework if people open the project in Visual Studio.